### PR TITLE
Remove redundant case details

### PR DESCRIPTION
**Pull request by tomxelliott**

### CODE REVIEW

File: src/functionalTest/java/uk/gov/hmcts/divorce/testutil/FunctionalTestSuite.java [hunk 1]
```diff
@@ -50,13 +50,13 @@
...
-            .petitionerLastName(TEST_LAST_NAME)
-            .divorceOrDissolution(DIVORCE)
-            .petitionerEmail(TEST_USER_EMAIL)
-            .build();
+    protected Map<String, Object> caseData() {
+        Map<String, Object> caseDataMap = new HashMap<>();
+        caseDataMap.put("petitionerFirstName", TEST_FIRST_NAME);
+        caseDataMap.put("petitionerLastName", TEST_LAST_NAME);
+        caseDataMap.put("divorceOrDissolution", DIVORCE);
+        caseDataMap.put("petitionerEmail", TEST_USER_EMAIL);
```
Comment (nitinprabhu) [inline src/functionalTest/java/uk/gov/hmcts/divorce/testutil/FunctionalTestSuite.java:58]: would be nice to use constants so you can refer everything from one place and if field name changes it reflects everywhere

Comment (tomxelliott) [reply src/functionalTest/java/uk/gov/hmcts/divorce/testutil/FunctionalTestSuite.java:58]: 👍

File: src/integrationTest/java/uk/gov/hmcts/divorce/solicitor/SolicitorCreateTest.java [hunk 2]
```diff
@@ -116,13 +117,13 @@ private String expectedResponse(final String jsonFile) throws IOException {
...
-            .divorceOrDissolution(DivorceOrDissolution.DIVORCE)
-            .divorceCostsClaim(YES)
-            .build();
+    private Map<String, Object> caseData() {
+        Map<String, Object> caseData = new HashMap<>();
+        caseData.put("petitionerFirstName", TEST_FIRST_NAME);
+        caseData.put("petitionerLastName", TEST_LAST_NAME);
+        caseData.put("petitionerEmail", TEST_USER_EMAIL);
+        caseData.put("divorceOrDissolution", DivorceOrDissolution.DIVORCE);
+        caseData.put("divorceCostsClaim", YES);
```
Comment (nitinprabhu) [inline src/integrationTest/java/uk/gov/hmcts/divorce/solicitor/SolicitorCreateTest.java:126]: there is `CaseFieldsConstants` class which you can use to define these constants(and through out where you are using keys of case data)

Comment (tomxelliott) [reply src/integrationTest/java/uk/gov/hmcts/divorce/solicitor/SolicitorCreateTest.java:126]: 👍

**Review by linusnorton — APPROVED** [review]

File: src/integrationTest/java/uk/gov/hmcts/divorce/solicitor/SolicitorCreateTest.java (hunk 2 above)
Comment (tomxelliott) [reply src/integrationTest/java/uk/gov/hmcts/divorce/solicitor/SolicitorCreateTest.java]: Have actioned this, thanks for the review @nitinprabhu

**Review by nitinprabhu — APPROVED** [review]

**Pull request merged**